coral-component-tablist/src/tests/test.TabList.js

```diff
@@ -563,8 +565,10 @@ describe('TabList', function() {
       
       setTimeout(() => {
         expect(el._elements.line.style.width).to.equal('');
-        expect(el._elements.line.style.height).to.not.equal('');
-        expect(el._elements.line.style.translate).to.not.equal('');
+        if (!IS_FIREFOX) {
```

Comment:
@majornista  What's up with Firefox here ?

Reply:
For some reason, the tests are failing for Firefox only in this unit test.